src/modules/assertions/require_cc0.ts

```diff
@@ -0,0 +1,30 @@
+import { context, getOctokit } from "@actions/github";
+import { setFailed } from "@actions/core";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+import _ from "lodash";
+
+// BOT will fail if no CC0 detected in PR body
+// Just after: Verify correct environment and request context in main.ts, if no CC0 is detected, 
+// BOT will fail and comment: "error: No CC0 waiver detected".
+
+export const  checkCC0 = () => {
+        
+	const github = getOctokit(GITHUB_TOKEN).rest;
+  const copyrightRegex = /^[\s\S]*\n## Copyright\nCopyright and related rights waived via \[CC0\]\(\.\.\/LICENSE.md\)\.\n?$/g
+  
+  // TODO: we should figure out what conditions allow us to reach this code with `body` being null.
+  // If body is null there is no license, EIP-BOT must exit(1). let me know if you agree with it..
```

Comment:
```suggestion
```
Questions like this should be in the PR discussion, rather in the code.  In this case, we should figure out what conditions (if any) can lead to a missing body and then decide whether it makes sense to exit(1) or just return.